java/src/Ice/src/main/java/com/zeroc/Ice/_ObjectPrxFactoryMethods.java

```diff
@@ -8,6 +8,15 @@
  */
 @SuppressWarnings("unchecked")
 public abstract class _ObjectPrxFactoryMethods<T extends ObjectPrx> extends _ObjectPrxI {
+  // TODO: delete this default constructor eventually.
+  public _ObjectPrxFactoryMethods() {
+    super();
+  }
+
+  public _ObjectPrxFactoryMethods(ObjectPrx obj) {
```

Comment:
Make it protected?

Reply:
I swear I did this at some point, but must of lost it in a `git reset`.
Done.